kata-deploy/scripts/kata-deploy.sh

```diff
@@ -87,7 +87,7 @@ function configure_crio() {
 	local kata_qemu_virtiofs_conf="crio.runtime.runtimes.kata-qemu-virtiofs"
 
 	# add kata-qemu config
-	if grep -q "\[$kata_qemu_conf\]" $crio_conf_file; then
+	if grep -q "$\[$kata_qemu_conf\]" $crio_conf_file; then
```

Comment:
uhmm, this didn't work for me, I have to use `egrep` or `grep -e`

Reply:
and `s/$/^/`

Reply:
doh, regex brainfart... yes, I meant `^`, of course.
wrt `-e` and `-E` ... looking at the kata deploy container and the gnu grep manual, I don't think we (strictly) need either in this script:
- the `-e` defines where the pattern string is, which is useful if you have multiple patterns on the command line, but we only have one, so I don't think we need it (`grep` defaults to reading one pattern anyway).
- the `-E` enables extended expressions, but the gnu grep manual says `In  GNU  grep there is no difference in available functionality between basic and extended syntaxes.`, so again, I don't think we need it.

Having said that, I guess to make the script more portable and explicit, sure, I'll extend the grep parameters to `-qEe` - I just tested that (a bit better than I apparently tested last time ;-), and it works fine.

thanks @devimc , good pickups. Let me go fix and push...